### PR TITLE
Php73 node12

### PR DESCRIPTION
**Pull request by nicolas-bastien**

### CODE REVIEW

File: builder/Dockerfile [hunk 1]
```diff
@@ -40,3 +39,8 @@ RUN sudo apt-get update &&  apt-get install yarn
 RUN apt-get autoremove -y --purge \
     && apt-get clean \
     && rm -Rf /tmp/*
+
+RUN useradd --shell /bin/bash -u 1000 -o -c "" -m dev
```
Comment (nicolas-bastien) [inline builder/Dockerfile]: @mathieu-ducrot I found I have that part uncommited so maybe this is why you add right conflict

**Comment by nicolas-bastien** [comment]
@mathieu-ducrot pushed on docker hub so you can test

File: builder/Dockerfile (hunk 1 above)
Comment (mathieu-ducrot) [reply builder/Dockerfile]: @nicolas-bastien i did not succed to reproduce the permission conflict that i had, but it's still ok to add this.

File: Dockerfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-FROM php:7.2-fpm
+FROM php:7.3-fpm
```
Comment (mathieu-ducrot) [inline Dockerfile]: @nicolas-bastien actually if i php --version on the latest pull of smartbooster/php-fpm:builder on smartpilot i still get this

> PHP 7.2.10 (cli) (built: Sep 15 2018 02:33:49) ( NTS )
> Copyright (c) 1997-2018 The PHP Group
> Zend Engine v3.2.0, Copyright (c) 1998-2018 Zend Technologies
>     with Zend OPcache v7.2.10, Copyright (c) 1999-2018, by Zend Technologies

File: Makefile [hunk 3]
```diff
@@ -2,13 +2,13 @@ login:
 	docker login
 
 build:
-	docker build -t smartbooster/docker-php-fpm .
-	docker build -t smartbooster/docker-php-fpm:builder ./builder
+	docker build -t smartbooster/php-fpm .
```
Comment (mathieu-ducrot) [inline Makefile]: smartbooster/php-fpm was already the name of the docker image or did you actually change it ?

File: builder/Dockerfile [hunk 4]
```diff
@@ -24,7 +23,7 @@ RUN apt-get install -y \
     gnupg \
     sudo
 
-RUN curl -sL https://deb.nodesource.com/setup_8.x | sudo bash -
+RUN curl -sL https://deb.nodesource.com/setup_12.x | sudo bash -
```
Comment (mathieu-ducrot) [inline builder/Dockerfile]: v12.13.1 on node --version so it's ok !

File: builder/Dockerfile [hunk 5]
```diff
@@ -40,3 +39,8 @@ RUN sudo apt-get update &&  apt-get install yarn
 RUN apt-get autoremove -y --purge \
     && apt-get clean \
     && rm -Rf /tmp/*
+
+RUN useradd --shell /bin/bash -u 1000 -o -c "" -m dev
+COPY .bashrc /home/dev/.bashrc
+RUN export HOME=/home/dev
```
Comment (mathieu-ducrot) [inline builder/Dockerfile]: @nicolas-bastien what is it for ?

File: Dockerfile (hunk 2 above)
Comment (nicolas-bastien) [reply Dockerfile]: @mathieu-ducrot I added a ssh command to validate and it's ok for me

Capture du 2019-12-29 08-22-51

I checked if I miss to push but it was ok, can you check ?

Comment (mathieu-ducrot) [reply Dockerfile]: @nicolas-bastien it's ok for the latest version but check it on the builder tag version : 

`docker run --rm -it smartbooster/php-fpm:builder bash` it prompt me a PHP 7.2.10

Comment (nicolas-bastien) [reply Dockerfile]: @mathieu-ducrot ok I've fixed it in next PR !

**Pull request merged**